lib/use-checkout-pricing.js

```diff
@@ -84,8 +80,8 @@ export default function useCheckoutPricing (initialInputs, handleError = throwEr
     };
 
     function addRestInputs(restInputs, checkoutPricing) {
-      const { PRICING_METHODS } = checkoutPricing;
-      const exclude = ['reset', 'remove', 'reprice', 'subscriptions', 'adjustments', 'addon', 'plan'];
+      const PRICING_METHODS = checkoutPricing.PRICING_METHODS || checkoutPricing.pricing.PRICING_METHODS;
```

Comment:
I wonder if this method should only expect either a CheckoutPricing or a PricingPromise(CheckoutPricing) in order to prevent having to look in both locations.

Reply:
I updated this so that `addRestInputs` is always called with a PricingPromise(CheckoutPricing) https://github.com/recurly/react-recurly/pull/57/commits/8e44df5726f10460841fefcca33f174a0e836483